### PR TITLE
VEN-91 | New Front Page

### PR DESCRIPTION
**Pull request by Nizar-Rahme**

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -11,17 +11,27 @@ import ThankYouPage from './pages/containers/ThankYouPageContainer';
...
+    locale={locale}
+    key={locale}
+    messages={messages[locale]}
+    defaultLocale={LocaleOpts.FI}
+  >
     <Switch>
-      <Redirect exact path={`/${localeParam}/`} to={`/${localeParam}/berths`} />
+      <Route exact path={`/${localeParam}`} component={FrontPage} />
       <Route exact path={`/${localeParam}/berths`} component={BerthPage} />
       <Route exact path={`/${localeParam}/selected_berths`} component={SelectedBerthPage} />
```
Comment (rBoost) [inline src/components/App.tsx]: Ugh... `selected_berths` looks really ugly, it should prolly be `selected-berths`

File: src/components/layout/KoroSection/index.tsx [hunk 2]
```diff
@@ -1,28 +1,65 @@
 import classnames from 'classnames';
```
Comment (rBoost) [inline src/components/layout/KoroSection/index.tsx]: Why is this file called as `index.tsx`? Would it be a huge thing to change as `KoroSection.tsx`?

File: src/components/layout/index.tsx [hunk 3]
```diff
@@ -1,72 +1,21 @@
-import classNames from 'classnames';
```
Comment (rBoost) [inline src/components/layout/index.tsx]: index.tsx?

**Comment by Nizar-Rahme** [comment]
I renamed a few `index.tsx` files to follow the components' names, I am determined to continue with rest in a separate pull request where there are no big changes in the logic so it will be easier to be reviewed.

**Pull request merged**